src/sources/forus-platform/pug/tpl/pages/transaction-bulk.pug

```diff
@@ -7,15 +7,26 @@
                 translate="page_state_titles.transactions")
             .breadcrumb-item.active(ng-bind="'#' + $ctrl.transactionBulk.id")
 
-        .card: .card-section: .card-title
-            strong(ng-bind="$ctrl.transactionBulk.voucher_transactions_amount | currency_format")
-            strong.text-primary.pull-right Totaal
-
         .card
-            .card-header: .card-title(translate="financial_dashboard_transaction.labels.details")
+            .card-header: .flex-row
+            
```

Comment:
should check if the transaction sum is also removed from provider dashboard; (cause it usefull there not on sponsor dashboard)